cv.typ

```diff
@@ -265,7 +265,7 @@
                 #if cert.url != none [
                     *#link(cert.url)[#cert.name]* \
```

Comment:
Please modify the PR in order to account for the other branch of the if-statement as well.

```suggestion
                    *#link(cert.url)[#cert.name]* #h(1fr) ID: #cert.id \
```

Also how will this look like for long certificate names along with the a long ID as well?